src/app/zap-templates/zcl/data-model/chip/onoff-cluster.xml

```diff
@@ -52,7 +52,7 @@ limitations under the License.
     <attribute side="server" code="0x4000" define="GLOBAL_SCENE_CONTROL" type="boolean" default="1"                 optional="true">GlobalSceneControl</attribute>
     <attribute side="server" code="0x4001" define="ON_TIME"              type="int16u"  default="0" writable="true" optional="true">OnTime</attribute>
     <attribute side="server" code="0x4002" define="OFF_WAIT_TIME"        type="int16u"  default="0" writable="true" optional="true">OffWaitTime</attribute>
-    <attribute side="server" code="0x4003" define="START_UP_ON_OFF"      type="enum8"               writable="true" optional="true">StartUpOnOff</attribute>
+    <attribute side="server" code="0x4003" define="START_UP_ON_OFF"      type="enum8"   default="0" writable="true" isNullable="true" optional="true">StartUpOnOff</attribute>
```

Comment:
We need tests for this (followup OK).  Specifically, we should have a YAML test where we set StartupOnOff to null, restart the server, and check that OnOff is whatever it was before the restart.

Reply:
Do we have already have something existent with the yaml to create a reset and start back from there?

Reply:
We have an existing "reboot" thing, yes.  See `src/app/tests/suites/TestSystemCommands.yaml` for some examples.  But briefly there are "Stop", "Start", and "Reboot" commands.  You may not need to pass the discriminator argument to Start/Reboot; I have not checked.